.github/workflows/ruby.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
     # change this to (see https://github.com/ruby/setup-ruby#versioning):
     # uses: ruby/setup-ruby@v1
-      uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
```

Comment:
I just use `ruby/setup-ruby@v1` like it says in the comment above to get automatic features and bug fixes. We can do the same instead of chasing versions.